### ISSUE TITLE
Crash on Image upload

### ISSUE THREAD

**Issue by padyx** [padyx/purplemoon-android#16]
```
#!java

java.lang.IncompatibleClassChangeError: interface not implemented
at org.apache.commons.io.IOUtils.closeQuietly(IOUtils.java:303)
at ch.defiant.purplesky.network.ContentUriRequestBody.<init>(ContentUriRequestBody.java:57)
at ch.defiant.purplesky.network.ContentUriRequestBody.create(ContentUriRequestBody.java:64)
```

**Comment by padyx** [comment]
(Original author of comment is Bitbucket user 'Chakotay')
(Correct milestone after renaming)

**Comment by padyx** [comment]
(Original author of comment is Bitbucket user 'Chakotay')
Fix Incompatible class change (Closable not implemented until Android 4.4). Resolve #108


→ <<cset 2c2df3ba5192>>

(legacy branch)

**Comment by padyx** [comment]
(Original author of comment is Bitbucket user 'Chakotay')
Fix Incompatible class change (Closable not implemented until Android 4.4). Resolve #108


→ <<cset 3f3f1ca58d21>>

**Comment by padyx** [comment]
(Original author of comment is Bitbucket user 'Chakotay')
Related issue: #100
